packages/shortest/src/browser/core/browser-tool.ts

```diff
@@ -39,64 +35,65 @@ export class BrowserTool extends BaseBrowserTool {
   private testContext?: TestContext;
   private readonly MAX_SCREENSHOTS = 10;
   private readonly MAX_AGE_HOURS = 5;
+  private mailosaurTool?: MailosaurTool;
+  private config!: ShortestConfig;
 
   constructor(
-    page: Page,
+    page: Page, 
     browserManager: BrowserManager,
-    config: BrowserToolConfig,
+    config: BrowserToolConfig
   ) {
     super(config);
     this.page = page;
     this.browserManager = browserManager;
-    this.screenshotDir = join(process.cwd(), ".shortest", "screenshots");
+    this.screenshotDir = join(process.cwd(), '.shortest', 'screenshots');
```

Comment:
Any linting changes should be done in a separate PR; otherwise, it increased the diff for this one, and it makes it harder to review.

Reply:
I fixed the linting errors in a different branch and merged main here. Autofix is passing!